src/client/src/commons/form/borehole/hydrogeology/waterIngressDisplay.js

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import { Card, Stack, Tooltip, Typography } from "@mui/material";
+import ModeEditIcon from "@mui/icons-material/ModeEdit";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { useTranslation } from "react-i18next";
+import { styled } from "@mui/system";
+
+const WaterIngressDisplay = props => {
+  const {
+    waterIngress,
+    setSelectedWaterIngress,
+    isEditable,
+    deleteWaterIngress,
+  } = props;
+  const { t, i18n } = useTranslation();
+
+  function timesToReadableDuration(startTime, endTime) {
+    const timeStampStart = new Date(startTime).getTime();
+    const timestampEnd = new Date(endTime).getTime();
+    const durationInMinutes = (timestampEnd - timeStampStart) / 60000;
+    if (durationInMinutes < 0) return "-";
+    const hours = Math.floor(durationInMinutes / 60);
+    const minutes = Math.floor(durationInMinutes % 60);
+    let result = "";
+    if (hours > 0) {
+      result +=
+        hours + (hours === 1 ? " " + t("hour") : " " + t("hours") + " ");
+    }
+    if (minutes > 0) {
+      result +=
+        minutes +
+        (minutes === 1 ? " " + t("minute") : " " + t("minutes") + " ");
```

Comment:
PP: Der Whitespace with in jedem Fall an den Minuten- bzw. Stunden-Text vorangestellt, daher kann man den Ausdruck leicht vereinfachen zu:
```
    if (minutes > 0) {
      result +=
        minutes + " " +
        (minutes === 1 ? t("minute") : t("minutes") + " ");
```

Q: Braucht es den Whitespace am Schluss nur wenn der Text in Plural ist oder immer? Falls es den Whitespace am Schluss immer braucht, könnte man auch mit String-Interpolation arbeiten, vielleicht macht es den Code dann klarer. Beispiel: 
```
    if (minutes > 0) {
      result += `{minutes} {(minutes === 1 ? t("minute") : t("minutes")} `;
```